contrib/epee/include/epee/net/connection_basic.hpp

```diff
@@ -50,6 +50,11 @@
 #include <utility>
 
 #include <boost/asio.hpp>
+#if BOOST_VERSION >= 108700
+namespace boost::asio {
+  typedef io_context io_service;
```

Comment:
I've never been able to read typedefs in the proper direction without stopping to think about it.  Perhaps:
```suggestion
  using io_service = io_context;
```

Reply:
(There are a bunch more in this PR as well)